src/components/client/pages/home/Atoms/HomeAtomButton.jsx

```diff
@@ -0,0 +1,11 @@
+import css from './HomeAtomButton.module.css';
+
+const HomeAtomButton = ({ text }) => {
+  return (
+    <button className={css.HomeAtomButton} type="button">
+      {text}
+    </button>
```

Comment:
Are we able to use our button component? ❓

Reply:
Of course, I can change the button as soon as it's available on dev.

Reply:
Isn't ❓

Reply:
It was enough to merge the 'dev' branch, and the button appeared.